packages/io-services-cms-models/src/service-lifecycle/index.ts

```diff
@@ -164,6 +168,7 @@ const FSM: FSM = {
           fsm: {
             state: "approved",
             approvalDate,
+            autoPublish: current.fsm.autoPublish ?? false,
```

Comment:
if is possibile, try to apply DRY principles....we avoid repeating `autopublish` default value 😉

Reply:
We added a function `getAutoPublish` in order to handle the default value for `autoPublish` globally.